### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by victhevenot**
codealong to teach hashes @jmburges @AnnJohn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,123 @@
...
++ Explain what a hash in JavaScript is
++ Create an object in JS
++ Access a value from an object
++ Add a key-value pair to an object
++ Delete a key-value pair from an object
++ Iterate over key-value pairs in an object
+
+## Intro 
+
+In JavaScript, all objects are effectively key-value pairs. We're not going to talk about objects in relation to Object Orientation, but objects as hashes. We will eventually get to objects and properties of objects, but right now we're just focused on how to create, manipulate and delete key-value pairs from a "hash".
```
Comment (ipc103) [inline README.md]: maybe add a little more to explain how this is different from Ruby. "JavaScript Objects behave sort of like a cross between Classes and Hashes in Ruby. For now, we're just going to focus on how to use them like a hash."

File: README.md [hunk 2]
```diff
@@ -1,7 +1,123 @@
...
+meals["breakfast"] = "oatmeal"
+meals["lunch"] = "turkey sandwich"
+meals["dinner"] = "steak and potatoes"
+```
+
+## Accessing A Value
+Just like in Ruby, we access the value of an object from it's key:
+
+```js
+var meals = {breakfast: "oatmeal", lunch: "turkey sandwich", dinner: "steak and potatoes"}
```
Comment (ipc103) [inline README.md:56]: at some point, note that there's no `=>` syntax?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,123 @@
...
+```js
+var meals = {breakfast: "oatmeal", lunch: "turkey sandwich", dinner: "steak and potatoes"};
+meals["breakfast"] = "oatmeal and banana";
+meals;
+//returns {breakfast: "oatmeal and banana", lunch: "turkey sandwich", dinner: "steak and potatoes"}
+```
+
+## Check Empty Object
+
+You can check to see if your object is empty by using `Object.keys(yourObject)` which returns an array of all the keys in your object.
```
Comment (ipc103) [inline README.md]: maybe note that JS doesn't have a nice `.empty?` method like Ruby?

File: README.md [hunk 4]
```diff
@@ -1,7 +1,123 @@
...
+  console.log("for " + key + " I ate " + meals[key]);
+}
+```
+
+
+## Instructions
+
++ Create an object that is stored in the variable `myPlaylist`. The key of this object will be the artists and the values will be the song titles.
+
++ Create a function `addToPlaylist` which accepts three parameters: the playlist (an object), a song (a string), and an artist (a string). The body of the function should add the song and artist as a key-value pair to the playlist object. The function should return the whole playlist.
```
Comment (ipc103) [inline README.md:126]: clarify which should be the key and which should be the value

**Comment by ipc103** [comment]
forked and cloned - tests look good! just some minor comments @vicfriedman

File: README.md (hunk 1 above)
Comment (victhevenot) [inline README.md]: added!

File: README.md (hunk 2 above)
Comment (victhevenot) [inline README.md:56]: done.

File: README.md (hunk 3 above)
Comment (victhevenot) [inline README.md]: done

**Pull request merged**